Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,15 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            const string name = "Mahdi";
+            const int age = 29;
+            const double weight = 80.9;
+            const float height = 166.6f;
+            Console.WriteLine($"{name} is {age} years old, his weight is {weight} kg and his height is {height} cm. ");
+            //BMI calculations
+            double bmi = weight * 703 / height / height;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add the 'const' modifier to 'bmi'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5139827)